idea/testData/inspectionsLocal/annotationTargetExpression/noRetention.kt

```diff
@@ -0,0 +1,4 @@
+// PROBLEM: none
+// DISABLE-ERRORS
```

Comment:
Looks like switching to `RESTRICTED_RETENTION_FOR_EXPRESSION_ANNOTATION_WARNING` should fix this.